### ISSUE TITLE
Wrong imports of edge-specific packages bundled with edge functions

### ISSUE THREAD

**Issue by Brendonovich** [magne4000/vite-plugin-vercel#12]
In attempting to write an edge function that uses `@vercel/edge-config`, I am getting build errors in Vercel:

```
Build Completed in /vercel/output [8s]
--
13:30:52.954 | Deploying outputs...
13:30:53.363 | Error: The Edge Function "edge-config" is referencing unsupported modules:
13:30:53.363 | - index.js: fs/promises
13:30:54.019 | Deployment completed
13:30:53.629 | NOW_SANDBOX_WORKER_EDGE_FUNCTION_UNSUPPORTED_MODULES: The Edge Function "edge-config" is referencing unsupported modules:
- index.js: fs/promises
```

I found in the `package.json` of  `@vercel/edge-config` that there is a specific import that should be used when bundlers are building it for the Edge Runtime - `edge-light`.
```json
"exports": {
  "types": "./dist/index.d.ts",
  "import": {
    "edge-light": "./dist/index.edge.js",
    "node": "./dist/index.node.js",
    "default": "./dist/index.edge.js"
  },
  "require": {
    "edge-light": "./dist/index.edge.cjs",
    "node": "./dist/index.node.cjs",
    "default": "./dist/index.edge.cjs"
  }
}
```

`vite-plugin-vercel` appears to be using the `node` export, causing `fs/promises` to be included. In `@vercel/node` it appears that they do a bunch of custom [file resolution](https://github.com/vercel/vercel/blob/6115f0d74a90b1931ce500be2c3a05645c8529f6/packages/node/src/index.ts#L178-L180) and transformation, idk if something similar would have to be done here or if you could just configure esbuild to use the `edge-light` export.

I've made a reproduction [here](https://github.com/Brendonovich/vite-plugin-vercel-edge-repro).

**Comment by magne4000** [comment]
It should indeed be possible to target `edge-light` with esbuild. I'll have some time next week to try that

**Comment by magne4000** [comment]
Fixed in `0.2.0` version :rocket:

Also your edge function should be a default export:

```diff
import { get } from "@vercel/edge-config";

-export async function GET() {
+export default async function GET() {
  const data = await get("config");

  return new Response(JSON.stringify(data));
}
```

**Comment by Brendonovich** [comment]
Oh whoops, I copied the function from a next 13 project, that's why it's a named export 😅 
Thanks for this!
